.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,87 @@
+name: Release PR
+on:
+  repository_dispatch:
+    types: [release-published]
+
+jobs:
+  release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+
+      - name: Cache 📦
+        uses: actions/cache@v3
+        with:
+          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Setup Node Environment ⬢
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Update version in package.json
```

Comment:
Wait, should we be using this in our other repos? This seems like it solves some problems we've had with trying to publish and update package.json based on a git tag?!

Reply:
I like that idea, It's a pretty simple and it could save a step

Reply:
Awesome, ok, let's add this to ts-template, and then also into js-tableland and friends. Seems like a nicer pattern than I was able to come up with!

Reply:
https://github.com/tablelandnetwork/js-template/issues/37